lib/main_publiccloud.pm

```diff
@@ -77,47 +77,68 @@ sub load_publiccloud_consoletests {
     loadtest 'console/libgcrypt';
 }
 
+sub load_released_publiccloud_tests {
+    if (get_var('PUBLIC_CLOUD_PREPARE_TOOLS')) {
+        loadtest "publiccloud/prepare_tools";
+    }
+    elsif (get_var('PUBLIC_CLOUD_IMG_PROOF_TESTS')) {
+        loadtest "publiccloud/img_proof";
+    }
+    elsif (get_var('PUBLIC_CLOUD_LTP')) {
+        loadtest 'publiccloud/run_ltp';
+    }
+    elsif (get_var('PUBLIC_CLOUD_SLES4SAP')) {
+        loadtest 'publiccloud/sles4sap';
+    }
+    elsif (get_var('PUBLIC_CLOUD_ACCNET')) {
+        loadtest 'publiccloud/az_accelerated_net';
+    }
+    elsif (get_var('PUBLIC_CLOUD_CHECK_BOOT_TIME')) {
+        loadtest "publiccloud/boottime";
+    }
+    elsif (get_var('PUBLIC_CLOUD_FIO')) {
+        loadtest 'publiccloud/storage_perf';
+    }
+    elsif (get_var('PUBLIC_CLOUD_CONSOLE_TESTS')) {
+        my $args = OpenQA::Test::RunArgs->new();
+        loadtest "publiccloud/prepare_instance", run_args => $args;
+        loadtest "publiccloud/register_system", run_args => $args;
+        loadtest "publiccloud/ssh_interactive_start", run_args => $args;
+        load_extra_tests_prepare();
+        load_publiccloud_consoletests();
+        loadtest("publiccloud/ssh_interactive_end", run_args => $args);
+    }
+    elsif (get_var('PUBLIC_CLOUD_IMAGE_LOCATION')) {
```

Comment:
in a fact **every** pubcloud test has this variable , so basically it was already the case - when something is broken we got plenty of `upload_img` tests runs which silently passing because nothing to upload . so we actually "hiding" problems . this is why I want to `upload_img` to start only if someone explicitly asked for this , https://gitlab.suse.de/qac/qac-openqa-yaml/-/merge_requests/505 .

Reply:
*this change is ready for review*